app/lib/common/utilities/pdf_preview.dart

```diff
@@ -38,6 +38,23 @@ class _PdfPreviewState extends State<PdfPreview> {
                 return PDFViewerScaffold(
                   appBar: AppBar(
                     title: Text('Example PDF'),
+                    actions: [
+                      OutlinedButton.icon(
+                          onPressed: () {
+                            Navigator.pop(context);
```

Comment:
I think this is dangerous because this is popping the navigator while the widget is still live. We should refactor this int he future.